doc/classes/SceneTree.xml

```diff
@@ -82,26 +83,25 @@
 				}
 				[/csharp]
 				[/codeblocks]
-				The timer will be automatically freed after its time elapses.
 			</description>
 		</method>
 		<method name="create_tween">
 			<return type="Tween" />
 			<description>
-				Creates and returns a new [Tween].
+				Returns a newly created [Tween].
 			</description>
 		</method>
 		<method name="get_first_node_in_group">
 			<return type="Node" />
 			<param index="0" name="group" type="StringName" />
 			<description>
-				Returns the first node in the specified group, or [code]null[/code] if the group is empty or does not exist.
+				Returns the first [Node] in the specified [param group], or [code]null[/code] if the group does not exist. See also [method get_nodes_in_group].
 			</description>
 		</method>
 		<method name="get_frame" qualifiers="const">
 			<return type="int" />
 			<description>
-				Returns the current frame number, i.e. the total frame count since the application started.
+				Returns how many frames have been processed, since the application started. This is [i]not[/i] elapsed time.
```

Comment:
Maybe expand the last sentence, since I am not sure if I understand what it means right now.

```suggestion
				Returns how many frames have been processed since the application started. This is [i]not[/i] elapsed time, since each frame can have a different delta time.
```

Reply:
It means it's not equal to the time since the application started, and should not be used to measure it.